test/e2e/plain_provisioner_test.go

```diff
@@ -61,7 +61,7 @@ var _ = Describe("plain provisioner bundle", func() {
 			By("creating the testing Bundle resource")
 			bundle = &rukpakv1alpha1.Bundle{
 				ObjectMeta: metav1.ObjectMeta{
-					Name: "olm-crds-valid",
+					GenerateName: "olm-crds-valid",
```

Comment:
I reverted the changes from #487 in `plain_provisioner_test.go` back to the previous `main` commit.

This test's purpose is to verify that a bundle with a non-existent provisioner class name never gets reconciled.